lib/linter-integration.js

```diff
@@ -31,13 +31,15 @@ class Linter {
     }
     this.linter.setMessages(messages.map(match => ({
       type: match.type || 'Error',
-      text: match.message || 'Error from build',
+      text: !match.message && !match.html_message ? 'Error from build' : match.message,
+      html: match.html_message,
```

Comment:
I don't think we should invoke the linter API incorrectly. Since `message` is the most used I think it should have precedence. So if both are set, don't send `html_message` to linter. Does that sound okay to you?